### PR TITLE
Add napari-picasso recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@krpandit, do you agree with being added as a maintainer?

**Comment by krpandit** [comment]
> We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.
> 
> You can read a bit more about conda-forge at https://conda-forge.org/#contribute
> 
> @krpandit, @jaimergp do you agree with being added as maintainers?

Yes I'll be a maintainer.

**Comment by jaimergp** [comment]
> do you agree with being added as maintainers?

Yup!

File: recipes/napari-picasso/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-picasso" %}
+{% set version = "0.0.0" %}
```
Comment (jaimergp) [inline recipes/napari-picasso/meta.yaml:2]: Probably the wrong version :)

Comment (goanpeca) [reply recipes/napari-picasso/meta.yaml:2]: https://pypi.org/project/napari-PICASSO/

Screen Shot 2022-06-10 at 10 21 58 AM

It is very very very alpha @krpandit  ?

Comment (krpandit) [reply recipes/napari-picasso/meta.yaml:2]: yea I forgot to update that line.

It's pretty alpha, there are few more features I need to add, but the very basics should work.

Comment (goanpeca) [reply recipes/napari-picasso/meta.yaml:2]: See https://github.com/nygctech/PICASSO/issues/1

Comment (goanpeca) [reply recipes/napari-picasso/meta.yaml]: This has been fixed :)

Thanks @krpandit !

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes

File: recipes/napari-picasso/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - magicgui
+    - napari
+    - numpy
+    - psutil
+    - python >=3.8
+    - qtpy
+
+test:
+  imports:
+    - napari_picasso
```
Comment (jaimergp) [inline recipes/napari-picasso/meta.yaml:32]:
```suggestion
    - napari_picasso
    - napari_picasso.main_widget
    - napari_picasso.options_widget
    - napari_picasso.sink_widget
    - napari_picasso.source_widget
    - mine.mine
    - picasso.picasso
    - picasso.nn_picaso
```

File: recipes/napari-picasso/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+    - python >=3.8
+  run:
+    - dask-core
+    - magicgui
+    - napari
+    - numpy
+    - psutil
+    - python >=3.8
+    - qtpy
```
Comment (jaimergp) [inline recipes/napari-picasso/meta.yaml:28]:
```suggestion
    - pytorch
```

File: recipes/napari-picasso/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```
Comment (jaimergp) [inline recipes/napari-picasso/meta.yaml:20]:
```suggestion
    - python >=3.8
    - setuptools-scm
```

File: recipes/napari-picasso/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+test:
+  imports:
+    - napari_picasso
+    - napari_picasso.main_widget
+    - napari_picasso.options_widget
+    - napari_picasso.sink_widget
+    - napari_picasso.source_widget
+    - mine.mine
+    - picasso.picasso
+    - picasso.nn_picaso
```
Comment (jaimergp) [inline recipes/napari-picasso/meta.yaml:41]:
```suggestion
    - picasso.nn_picasso
```

**Pull request merged**